lib/broadway/rate_limiter.ex

```diff
@@ -53,6 +61,19 @@ defmodule Broadway.RateLimiter do
     {:ok, state}
   end
 
+  @impl true
+  def handle_call({:update_rate_limiting, opts}, _from, state) do
```

Comment:
Would you prefer this to be a cast here? There's not much difference and no errors are produced here, so I think a cast could work.

Reply:
This is good IMO.